### DISCUSSION TITLE
Smart routing on missing page

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#7406] [Feature Requests]
When you navigate to a missing page, use a `ref` link`, or have an invalid route saved in your `directus_users.last_page`... instead of a 404 page, I think it would be better to navigate to a default page and show a system notification about that previous page being missing. No need to show a useless page and force them to re-route.

This ties into https://github.com/directus/directus/discussions/4590, which would configure a default page for users.

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
